app/exercises/five.py

```diff
@@ -65,41 +71,112 @@ class CustomToolbar(NavigationToolbar2Tk):
 
         self.toolbar = CustomToolbar(self.canvas, frame)
         self.toolbar.update()
-        
+
         self.canvas._tkcanvas.pack(side=tk.TOP, fill=tk.BOTH, expand=True)
```

Comment:
As I remember there is method for getting this field and this **_tkcanvas** is pretty private as I can see